packages/server/lib/open_project.ts

```diff
@@ -200,14 +191,14 @@ export class OpenProject {
     return browsers.close()
   }
 
-  closeOpenProjectAndBrowsers () {
+  closeOpenProjectAndBrowsers (shouldCloseBrowser = true) {
     this.projectBase?.close().catch((e) => {
       this._ctx?.logTraceError(e)
     })
 
     this.resetOpenProject()
 
-    return this.closeBrowser()
+    return shouldCloseBrowser ? this.closeBrowser() : Promise.resolve()
```

Comment:
We don't want to close the browser if we're just modify the config file. There's probably some circumstances we'd need to relaunch but for now this is probably best.